class DataError(Exception):
    """Some data was not as expected."""
